### PR TITLE
Python. Homework #5. Вяткин Артем

### PR DESCRIPTION
**Pull request by DartVart**

### CODE REVIEW

File: homeworks/homework5/task1/treap_node.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        return left, right
+
+    def _get_without_largest(self) -> Optional["TreapNode[K, P]"]:
+        """Returns the subtreap without the largest key"""
+
+        if self.right is not None:
+            self.right = self.right._get_without_largest()
+            return self
+
+        # TODO: return self.left if self.left is not None else None
```
Comment (nbirillo) [inline homeworks/homework5/task1/treap_node.py]: Is it ok that you have this todo?

**Review by nbirillo — CHANGES_REQUESTED** [review]
Great! Thanks a lot for the type annotations and comments. I have a small comment

File: homeworks/homework5/task1/treap_node.py (hunk 1 above)
Comment (DartVart) [reply homeworks/homework5/task1/treap_node.py]: Oops, I forgot myself :)

**Review by nbirillo — APPROVED** [review]
Great!

**Pull request merged**